packages/web/src/pages/artist-dashboard-page/ArtistDashboardPage.module.css

```diff
@@ -159,3 +159,74 @@
   justify-content: stretch;
   margin: 12px 0px 36px;
 }
+
+.usdcContainer {
+  width: 100%;
+  flex-direction: column;
+  border-radius: var(--unit-3);
+  background: var(--static-white);
+  overflow: hidden;
+}
+
+.backgroundBlueGradient {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  gap: var(--unit-4);
+  padding: var(--unit-6);
+  background: linear-gradient(
+      315deg,
+      rgba(0, 0, 0, 0.1) 0%,
+      rgba(255, 255, 255, 0.1) 100%
+    ),
+    #2775ca;
+}
+
+.usdcTitleContainer {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  text-transform: uppercase;
+  font-weight: 900;
+  font-size: var(--font-5xl);
+  color: var(--static-white);
+}
+
+.usdcTitle {
+  display: flex;
+  gap: var(--unit-2);
+}
+
+.usdc {
+  opacity: 0.8;
+}
+
+.usdcInfo {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  color: var(--static-white);
```

Comment:
Same question here. We shouldn't need font properties or color in CSS anymore with the Text component. Unless it's truly a one-off